apps/discovery-platform/src/registered-node/index.spec.ts

```diff
@@ -169,7 +169,7 @@ describe("test registered node functions", function () {
   it("should keep updated_at updated", async function () {
     await createRegisteredNode(dbInstance, mockNode("1"));
     const node = await getRegisteredNode(dbInstance, "1");
-    await wait(1);
+    await wait(100);
```

Comment:
So what's the actual problem here?

Does wait not work correctly? Or why are the timestamps not different with 1ms difference?
Whats the granularity of the db timestamps?

Reply:
`wait` uses `setTimeout` which is not entirely accurate [mentioned in this section](https://nodejs.org/dist/latest-v17.x/docs/api/timers.html#settimeoutcallback-delay-args) - so relying on just 1ms wait could lead to this problem
DB granularity is to the millisecond

Reply:
Ok can we do with `wait(2)` then?

Reply:
makes sense, should not be a problem